gpMgmt/bin/gpinitsystem

```diff
@@ -1373,6 +1377,7 @@ CREATE_SEGMENT () {
 			# SEGMENT_INFO: ~ separated value, ex: host1~port1~data_dir~dbid1~contentid
 			# PAIR_SEGMENT_INFO: corresponding pair, ex: host2~port2~data_dir~dbid2~contentid
 			# INST_COUNT: Number of parallel script count
+			export PRELOAD_LIBRARIES
```

Comment:
Can we pass it as an argument to make the operation idempotent?

Reply:
There is an argument for gpinitsystem. But leak an argument for `make create-demo-cluster`

Reply:
I'm afraid exporting additional environment variables is making things complex. What do you think of adding an additional argument `PRELOAD_LIBRARIES` to the make command. E.g., `make PRELOAD_LIBRARIES=data_invert create-demo-cluster`

Reply:
there are 3 dataflows:

1. initdb to postgresql: using GP_PRELOAD_LIBRARIES env
2. gpinitsystem to gpcreateseg.sh: using PRELOAD_LIBRARIES env (renamed to GP_PRELOAD_LIBRARIES now)
3. from `make create-demo-cluster` to gpinitsystem: hard-coded for now.

I will add some like `GP_PRELOAD_LIBRARIES=data_invert make create-demo-cluster` after I fix all error in `make installcheck`